### DISCUSSION TITLE
no zeek/suricata target then why there are config files of suricata and zeek rule files in the directory?

### DISCUSSION THREAD

**Discussion by IamTenacious** [SigmaHQ/sigma#2793] [Q&A] [answered]
I see no target for zeek and suricata rules in the **sigmac -h output** then what the config files of suricata [https://github.com/SigmaHQ/sigma/blob/master/tools/config/ala-suricata.yml]() and zeek files are [](https://github.com/SigmaHQ/sigma/tree/master/rules/network/zeek) doing in the directory?

**Comment by frack113** [comment, marked as answer]
Sigma is not designed to make zeek or suricata rules.
So there is no backend.

On the other hand there are rules on the alert logs of zeek or siricata.
Like the antivirus rules. The rule does not detect the virus but the alert message of the antivirus.
[ala-suricata](https://github.com/SigmaHQ/sigma/blob/master/tools/config/ala-suricata.yml)  is for mapping the suricata **alert** log fields to Azure Log Analytics.
So you can make a rule like "if I see a alert.signature like '*RDP*' in the surica alert then i wake up the SOC with a critical alert" 😃 

I hope I made myself clear
